Woof/Woof/Model/Walking/Ratings.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/**
+ The ratings given by both the pet owner and the pet sitter
+ Conforms to the `Codable` protocols.
+ */
+struct Ratings: Codable {
```

Comment:
```suggestion
struct Rating: Codable {
```
We seemed discussed that we should use singular.

Reply:
Fixed with 7f70e76